src/modules/processing/services/vimeo.js

```diff
@@ -81,8 +88,10 @@ export default async function(obj) {
             id: obj.id,
             title: fileMetadata.title,
             author: fileMetadata.artist,
-            resolution: `${bestVideo["width"]}x${bestVideo["height"]}`,
-            qualityLabel: `${resolutionMatch[bestVideo["width"]]}p`,
+            resolution: `${bestVideo.width}x${bestVideo.height}`,
+            qualityLabel: `${
+                resolutionMatch[bestVideo.width] || bestVideo.width
```

Comment:
shouldn't fallback be `bestVideo.height`? resolution names are usually the height, current state would lead to something like 1920p (which makes no sense)